src/catalog/abstract_catalog.cpp

```diff
@@ -37,6 +38,8 @@
 #include "storage/storage_manager.h"
 #include "storage/table_factory.h"
 
+using std::make_shared;
```

Comment:
Don't use these `using` directives.

Reply:
@pmenon thanks! just pushed these changes. I have avoided the use of `using` directives in other projects but noticed that the codebase has a fair amount of these. I will do my best to refactor them out in the future for other diffs.